packages/clay-drop-down/src/Menu.tsx

```diff
@@ -96,17 +95,19 @@ const ClayDropDownMenu = React.forwardRef<HTMLDivElement, IProps>((
 	});
 
 	return (
-		<ClayPortal>
-			<div
-				{...otherProps}
-				className={classNames('dropdown-menu', className, {
-					'dropdown-menu-indicator-end': hasRightSymbols,
-					'dropdown-menu-indicator-start': hasLeftSymbols,
-					show: active,
-				})}
-				ref={ref}
-			>
-				{children}
+		<ClayPortal subPortalRef={subPortalRef}>
+			<div ref={subPortalRef}>
```

Comment:
Is this element necessary or can we just use the existing `ref`? If ref doesn't exist, we could have a default ref.

Reply:
Actually we can, but just I created another `ref` so that people do not have the `ref` of the div but the element... maybe it is something they do not expect.

Reply:
I think it might be best to use the existing `ref`, mostly so we don't have to have an extra div everywhere.

Reply:
Oh I guess I got it wrong, we need this extra div here because it will contain the nested portals inside if we use `dropdown-menu` as a sub portal it will break because the class has `overflow: hidden`.